node-upstream-tests/run_tests.js

```diff
@@ -0,0 +1,387 @@
+'use strict'
+
+/* eslint-disable no-console */
+
+const promisify = require('util').promisify
+const axios = require('axios')
+const childProcess = require('child_process')
+const fs = require('fs')
+const glob = promisify(require('glob'))
+const path = require('path')
+
+const NODE_BIN = process.env['NODE_BIN'] || '/usr/bin/node'
```

Comment:
Why specify this instead of using whatever default instance is available?

Reply:
I was thinking that it would be useful to be able to point to a specific instance, to run a specific version of node which might not be the default one, with a specific version of the repo. Of course on CI this is not really a problem